metals/src/main/scala/scala/meta/internal/metals/MetalsSymbolSearch.scala

```diff
@@ -40,7 +41,14 @@ class MetalsSymbolSearch(
       symbol: String,
       parents: ParentSymbols,
   ): Optional[SymbolDocumentation] =
-    docs.documentation(symbol, parents)
+    documentation(symbol, parents, ContentType.MARKDOWN)
+
+  override def documentation(
+      symbol: String,
+      parents: ParentSymbols,
+      contentType: ContentType,
```

Comment:
Could we not have this as a parameter to MetalsSymbolSearch to avoid the roundabout way? It's only set once, no?

Reply:
It's also used for `signatureHelp`, which has a totally separate capability https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#signatureHelpClientCapabilities.